_data/jobs.yml

```
@@ -1,3 +1,8 @@
+- expires: 2022-03-09
+  location: Michigan State University, East Lansing, MI
+  name: Cloud software engineer
```

Should we maybe have consistency in our capitalization?
```suggestion
  name: Cloud Software Engineer
```
Just so it looks on par / equal to the other roles on the site, which tend to capitalize (even if not grammatically correct)